helix-loader/src/lib.rs

```diff
@@ -3,14 +3,30 @@ pub mod grammar;
 
 use etcetera::base_strategy::{choose_base_strategy, BaseStrategy};
 use std::path::{Path, PathBuf};
+use std::sync::RwLock;
 
 pub const VERSION_AND_GIT_HASH: &str = env!("VERSION_AND_GIT_HASH");
 
+static CWD: once_cell::sync::Lazy<RwLock<PathBuf>> = once_cell::sync::Lazy::new(|| {
```

Comment:
You don't need the `Lazy` here just use an RwLock<Option<...>>>` instead . You only access the global in the wrapper function anyway so you can simply fill the option there

Reply:
Ah yeah, that's a nice way to avoid that `Lazy`. Updated the PR, not sure in the best way but let me know !